docs/guide/built-in-functions.md

```diff
@@ -74,34 +74,34 @@ Total number of functions: **{{ $page.functionsCount }}**
+| DAY              | Returns the day of the given date value.                                                    | DAY(Number)                                         |
+| DAYS             | Calculates the difference between two date values.                                          | DAYS(Date2; Date1)                                  |
+| DAYS360          | Calculates the difference between two date values in days, in 360-day basis.                | DAYS360(Date2; Date1[; Format])                     |
+| EDATE            | Shifts the given startdate by given number of months.                                       | EDATE(Startdate; Months)                            |
+| EOMONTH          | Returns the date of the last day of a month which falls months away from the start date.    | EOMONTH(Startdate; Months)                          |
+| HOUR             | Returns hour component of given time.                                                       | HOUR(Time)                                          |
+| INTERVAL         | Returns interval string from given number of seconds.                                       | INTERVAL(Seconds)                                   |
+| ISOWEEKNUM       | Returns an ISO week number that corresponds to the week of year.                            | ISOWEEKNUM(Date)                                    |
+| MINUTE           | Returns minute component of given time.                                                     | MINUTE(Time)                                        |
+| MONTH            | Returns the month for the given date value.                                                 | MONTH(Number)                                       |
+| NETWORKDAYS      | Returns the number of working days between two given dates.                                 | NETWORKDAYS(Date1; Date2[; Holidays])               | 
+| NETWORKDAYS.INTL | Returns the number of working days between two given dates.                                 | NETWORKDAYS.INTL(Date1; Date2[; Mode [; Holidays]]) |
+| NOW              | Returns current date + time.                                                                | NOW()                                               |
+| SECOND           | Returns second component of given time.                                                     | SECOND(Time)                                        |
+| TIME             | Calculates time from given hour, minute and second.                                         | TIME(Hour; Minute; Second)                          |
+| TIMEVALUE        | Interprets string as time.                                                                  | TIMEVALUE(Timestring)                               |
+| TODAY            | Returns an integer representing the current date as a number of full days since 1900-01-01. | TODAY()                                             |
```

Comment:
Looks good, however, I was hoping for clarifications about the return value in other date and time functions. WDYT?

Reply:
I added the return value descriptions for the other function that return dates. WDYT?

Reply:
Looks good, thanks!